docs/source/API/core/builtinreducers/Prod.rst

```diff
@@ -0,0 +1,99 @@
+``Prod``
+========
+
+.. role::cpp(code)
+    :language: cpp
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Specific implementation of `ReducerConcept <ReducerConcept.html>`_ performing a ``multiply`` operation
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
+
+    T result;
+    parallel_reduce(N,Functor,Prod<T,S>(result));
+
+Synopsis 
+--------
+
+.. code-block:: cpp
+        
+    template<class Scalar, class Space>
+    class Prod{
+        public:
+            typedef Prod reducer;
+            typedef typename std::remove_cv<Scalar>::type value_type;
+            typedef Kokkos::View<value_type, Space> result_view_type;
+            
+            KOKKOS_INLINE_FUNCTION
+            void join(value_type& dest, const value_type& src) const;
+
+            KOKKOS_INLINE_FUNCTION
+            void init(value_type& val) const;
+
+            KOKKOS_INLINE_FUNCTION
+            value_type& reference() const;
+
+            KOKKOS_INLINE_FUNCTION
+            result_view_type view() const;
+
+            KOKKOS_INLINE_FUNCTION
+            Prod(value_type& value_);
+
+            KOKKOS_INLINE_FUNCTION
+            Prod(const result_view_type& value_);
+    };
+
+Public Class Members
+--------------------
+
+Typedefs
+~~~~~~~~
+
+* ``reducer``: The self type.
+* ``value_type``: The reduction scalar type.
+* ``result_view_type``: A ``Kokkos::View`` referencing the reduction result 
+
+Constructors
+~~~~~~~~~~~~
+ 
+.. cppkokkos:kokkosinlinefunction:: Prod(value_type& value_);
+
+* Constructs a reducer which references a local variable as its result location.  
+ 
+.. cppkokkos:kokkosinlinefunction:: Prod(const result_view_type& value_);
+
+* Constructs a reducer which references a specific view as its result location.
+
+Functions
+~~~~~~~~~
+
+.. cppkokkos:kokkosinlinefunction:: void join(value_type& dest, const value_type& src) const;
+
+* Multiply ``src`` and ``dest`` and store in ``dest``:  ``dest*=src;``. 
+
+.. cppkokkos:kokkosinlinefunction:: void init(value_type& val) const;
+
+* Initialize ``val`` using the Kokkos::reduction_identity<Scalar>::prod() method. The default implementation sets ``val=1``.
```

Comment:
Please make `Kokkos::reduction_identity` a cross-reference to the class or at least monospace